dangvuthang/rocodingtest
# Feat/integrate backend register api

### Changelog

- Add `cors` on backend to handle cross-origin request
- Add `/face-taken` page for future use
- Update `sign in` logic

### How to test

- `checkout` this branch
- Run `npm install` on both **frontend** and **backend** folder
- Ensure that  file `.env.local` exists on frontend and file `.env` exists on backend
- Run `npm run dev` on both **frontend** and **backend**
- Navigate to `localhost:3000` and click on `login`

### Expected result

- frontend app will navigate to `/face-taken` page for the next step
- To ensure the data can be saved in the db correctly. Uncomment line 22 in `SignInButton.tsx` component

![Screenshot (578)](https://user-images.githubusercontent.com/43143636/161426298-d1765049-254a-4e9b-a2d2-fd1a9955120a.png)
